src/filters/string.rs

```diff
@@ -209,4 +273,44 @@ mod tests {
             assert_eq!(result.unwrap(), to_value(expected));
         }
     }
+
+    #[test]
+    fn test_title() {
+        let tests = vec![
+            ("foo bar", "Foo Bar"),
+            ("foo\tbar", "Foo\tBar"),
+            ("foo  bar", "Foo  Bar"),
+            ("f bar f", "F Bar F"),
+            ("foo-bar", "Foo-Bar"),
+            ("FOO\tBAR", "Foo\tBar"),
+            ("foo (bar)", "Foo (Bar)"),
+            ("foo (bar) ", "Foo (Bar) "),
+            ("foo {bar}", "Foo {Bar}"),
+            ("foo [bar]", "Foo [Bar]"),
+            ("foo <bar>", "Foo <Bar>"),
+            ("  foo  bar", "  Foo  Bar"),
+            ("\tfoo\tbar\t", "\tFoo\tBar\t"),
+            ("foo bar ", "Foo Bar "),
+            ("foo bar\t", "Foo Bar\t")
+        ];
+        for (input, expected) in tests {
+            let result = title(to_value(input), HashMap::new());
+            assert!(result.is_ok());
+            assert_eq!(result.unwrap(), to_value(expected));
+        }
+    }
+
+    #[test]
+    fn test_striptags() {
+        let tests = vec![
+            (r"<b>Joel</b> <button>is</button> a <span>slug</span>", "Joel is a slug"),
+            (r#"<p>just a small   \n <a href="x"> example</a> link</p>\n<p>to a webpage</p><!-- <p>and some commented stuff</p> -->"#,
+            "just a small example link to a webpage"),
```

Comment:
if you can add the tests from https://github.com/django/django/blob/ecb59cc6579402b68ddfd4499bf30edacf5963be/tests/utils_tests/test_html.py#L68-L88, it would be nice